pep517/check.py

```diff
@@ -200,4 +200,5 @@ def ansi(s, attr):
 
 
 if __name__ == '__main__':
+    print(ansi('pep517.check is deprecated. Consider switching to https://pypi.org/project/build/', 'red'))
```

Comment:
And, assuming we do change that, this should also not be wrapped in ansi red IMO